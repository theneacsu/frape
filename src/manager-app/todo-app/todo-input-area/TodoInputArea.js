import React from 'react'
import TodoForm from '../todo-form/TodoForm'

const TodoInputArea = () => (
  <div>
    <TodoForm placeholder="ex: Go for a run" buttonText="Add todo" />
  </div>
)

export default TodoInputArea
