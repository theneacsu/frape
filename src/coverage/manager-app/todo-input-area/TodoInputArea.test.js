import React from 'react'
import { shallow } from 'enzyme'
import TodoInputArea from '../../../manager-app/todo-app/todo-input-area/TodoInputArea'

test('Should render correctly the TodoInputArea component', () => {
  const wrapper = shallow(<TodoInputArea />)
  expect(wrapper).toMatchSnapshot()
})
