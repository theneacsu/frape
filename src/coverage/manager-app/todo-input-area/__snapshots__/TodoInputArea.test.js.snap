// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render correctly the TodoInputArea component 1`] = `
<div>
  <TodoForm
    buttonText="Add todo"
    placeholder="ex: Go for a run"
  />
</div>
`;
